### DISCUSSION TITLE
Cant set window title just follow the basic tutorial

### DISCUSSION THREAD

**Discussion by OriLiMu** [kovidgoyal/kitty#7608] [General]
I follow the tutorial [here](https://sw.kovidgoyal.net/kitty/remote-control/#tutorial).  And try to set the window title like this. 
But the window title not chanaged why? 

I use the command `kitten @ set-window-title --match cmdline:cat "new-title" ` which will work if you change the command to `kitten @ send-text --match cmdline:cat Hello, World` 
![image](https://github.com/kovidgoyal/kitty/assets/11295355/404ea500-b43b-4ea9-969c-a4b5df7c4b08)

**Comment by kovidgoyal** [comment]
it works fine. Change the active window to the one with cat running in it and you will see the title of the *OS Window* changing to match it.
